### PR TITLE
add support for django safedelete - undelete signal

### PR DESCRIPTION
**Pull request by MKaras93**

### CODE REVIEW

File: track_history/tracker.py [hunk 1]
```diff
@@ -59,8 +64,14 @@ def store_current_state(self, record_type=None):
         instance_deleted: bool = (
             record_type == self.history_record_model.RECORD_TYPES.deleted
         )
+        instance_undeleted: bool = (
```
Comment (akszydelko) [inline track_history/tracker.py:67]: H: Looks like the `instance_undeleted` variable is not used, so it's probably not needed

File: track_history/tracker.py [hunk 2]
```diff
@@ -117,7 +132,10 @@ def signal_receiver(self, instance, signal, **kwargs):
             DJANGO_SAFEDELETE_INSTALLED
             and signal == post_save
             and not created
-            and kwargs.get("update_fields", []) == ["deleted"]
+            and (
+                kwargs.get("update_fields", [])
+                == ["deleted"]
+            )
```
Comment (akszydelko) [inline track_history/tracker.py:138]: Q: Did black or something reformatted it? Previous version looked quite ok 🤔

File: track_history/tracker.py (hunk 1 above)
Comment (MKaras93) [reply track_history/tracker.py]: Good catch, removed.

File: track_history/tracker.py (hunk 2 above)
Comment (MKaras93) [reply track_history/tracker.py]: Not sure, fixed :P

**Pull request merged**